docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -27,6 +27,7 @@ The full list of `ChatUI` properties you can override include:
 * `channelNameFormatter`: Allows customizing the way channel names are formatted.
 * `messagePreviewFormatter`: Allows to generate a preview text for the given message.
 * `dateFormatter`: Allows changing the way dates are formatted.
+* `showDateSeparatorInEmptyThread`: Allows showing the DateSeparator item in the MessageListView, when opening an empty thread. This helps implement more complex use cases for empty threads and `ThreadPlaceholderItem`s.
```

Comment:
If we're thinking about `DateSeparatorItem` maybe it's best to put it down as such or alternatively just "date separator item"

```suggestion
* `showDateSeparatorInEmptyThread`: Allows showing the DateSeparator item in the `MessageListView`, when opening an empty thread. This helps implement more complex use cases for empty threads and `ThreadPlaceholderItem`s.
```